katana/katana.py

```diff
@@ -91,6 +93,208 @@ def readconfig():
+                                pyfiles = str(action_file).encode('utf-8', 'ignore')
+
+            py_files = str(pyfiles).encode('utf-8', 'ignore')
+            (_, file_name) = py_files.rsplit('/', 1)
+            path = str(gpysrcdir).encode('utf-8', 'ignore')
+            sys.path.append(path)
+            file_name = file_name.split('.')[0]
+            py_files_1 = py_files.split('/')
+            py_files_1 = ('.').join(py_files_1)
+            for action in actions: # If multiple action directories available in a action directory
+            # checking whether the action package is available in .py file name.
+                if action.strip() in py_files_1:
+                    actions_package = action.strip() + "." + file_name
+            modules = importlib.import_module(actions_package)# To get the class name for the action
+            # .py file in which the sub keyword is available
+            class_list_new = inspect.getmembers(modules, inspect.isclass)
+            for elem in class_list_new:
+                if actions_package in str(elem[1]):
+                    class_list_new = elem[0]
+            import_action_list.append('from ' + action.strip() + "." + file_name + " " +
+                                      "import " + class_list_new)
+            object_list = class_list_new + "_obj = " + class_list_new + "()"# Forming object
+            # creation and appending it to a list
+            object_list_new.append(object_list)
+            class_list = (object_list.split('=')[0]).strip()
+            Arguments = values.find('Arguments')# Finding all the arguments and it's values for
+            # each sub-keyword
+            if Arguments is not None:
+                argument_list = Arguments.findall('argument')
+                if len(argument_list) == 1:
+                    for argument in argument_list:
+                        argument_name = argument.get('name')
+                        if argument_name is not None:
+                            arg_value = argument.get('value')
+                            if arg_value is None or arg_value is False:
+                                arg_value = argument.text
+                            arg_name = argument_name + "=" + "\"" + arg_value + "\""
+                            kw_list_1.append("{0}.{1}({2})".format(class_list, kw_list, arg_name))
+                            # Forming doc string for wrapper keyword by using sub keyword details
+                            doc_string_value.append("The keyword {0} in Driver {1} has a defined "
+                                                    "arguments {2} You must want to send other "
+                                                    "values through data file".format(kw_list,
+                                                                                      driver,
+                                                                                      arg_name))
```

Comment:
This string is used in multiple places. Define this in the beginning of the function and use it when required with dynamic substitution.

Reply:
@priyaarun23 are the code related comments addressed? Would like to know if I should proceed to review again